internals/overlord/overlord.go

```diff
@@ -57,6 +57,9 @@ var (
 
 	// Hold no more than 1000 completed notices, even if they have not yet expired
 	pruneMaxNotices = 1000
+
+	// Hold	no more than 100 serviceData.
+	pruneMaxServiceData = 100
```

Comment:
Perhaps we should include the word "inactive" in the name, as that's what it actually is:
```suggestion
	pruneMaxInactiveServices = 100
```